infra/email_receipt_inbox/lambdas/parsers/parse_doordash.py

```
@@ -0,0 +1,284 @@
+
+    def handle_data(self, data):
+        if not self._skip:
+            self.chunks.append(data)
+
+
+def html_to_lines(html):
+    p = _TextExtractor()
+    p.feed(html)
+    text = "".join(p.chunks)
+    # kill zero-width / soft-hyphen / nbsp preheader junk
+    text = re.sub(r"[​‌‍­͏﻿]", "", text)
```

Ruff (0.15.21)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

(PLE2515)